### PR TITLE
(DOCSP-18425): Add Go to Driver Support for DBRefs table

### PR DESCRIPTION
**Pull request by jeff-allen-mongo**

### CODE REVIEW

**Review by matthewdale — APPROVED** [review]
Looks good 👍

File: source/reference/database-references.txt [hunk 1]
```diff
@@ -227,6 +227,12 @@ Driver Support for DBRefs
 
      - You can traverse references manually.
 
+   * - Go
```
Comment (matthewdale) [inline source/reference/database-references.txt:230]: Should these be in alphabetical order? If so, this should be above the "Haskell" line.

**Review by matthewdale — COMMENTED** [review]
Just gave this a 👍 , but I actually have one question about table ordering.

File: source/reference/database-references.txt (hunk 1 above)
Comment (jeff-allen-mongo) [reply source/reference/database-references.txt:230]: Oh my mistake you're right! Will fix. Cheers.

**Pull request merged**